source/features/add-co-authored-by.tsx

```diff
@@ -49,7 +75,7 @@ function addCoAuthors() {
 
 	addendum.delete(getOP());
 
-	field.value += '\n\n' + [...addendum.values()].join('\n');
+	field!.value += '\n\n' + [...addendum.values()].join('\n');
```

Comment:
Why do you need `!` here too? You already used it on line 65.

Reply:
Forgot to take it out. It was there originally. Will remove.

Reply:
Done.